integrations/gitlab/pyproject.toml

```diff
@@ -10,7 +10,7 @@ aiofiles = "^0.6.0"
 python-gitlab = "^3.14.0"
 pathlib = "^1.0.1"
 jsonschema = "^4.17.3"
-port-ocean = {version = "0.1.1", extras = ["cli"]}
+port-ocean = {version = "0.1.2rc3", extras = ["cli"]}
```

Comment:
are you sure?

Reply:
will be change after this pr
i need to publish 0.1.2